tests/integration/cqlengine/operators/test_where_operators.py

```diff
@@ -48,3 +59,56 @@ def test_operator_rendering(self):
         self.assertEqual("<=", six.text_type(LessThanOrEqualOperator()))
         self.assertEqual("CONTAINS", six.text_type(ContainsOperator()))
         self.assertEqual("LIKE", six.text_type(LikeOperator()))
+
+
+class TestIsNotNull(BaseCassEngTestCase):
+    def test_is_not_null_to_cql(self):
```

Comment:
nit: Could this be a unit test?

Reply:
It definitely can, but we don't have unit tests for cqlengine, so I figured at some point we didn't want to create a new folder just for a few tests. They don't start C* so they don't add more time to the run for being there.
If we moved them to unit I'd do so in a future PR

Reply:
:+1: awesome. filed a low-priority ticket: https://datastax-oss.atlassian.net/browse/PYTHON-975